src/strategies/subgraph-split-delegation/index.ts

```diff
@@ -0,0 +1,132 @@
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest, getScoresDirect } from '../../utils';
+import { Strategy } from '@snapshot-labs/snapshot.js/dist/src/voting/types';
+
+export const author = 'aragon';
+export const version = '0.1.0';
+
+const DEFAULT_BACKEND_URL =
+  'https://api.studio.thegraph.com/query/87073/split-delegation/version/latest';
+
+type Params = {
+  subgraphUrl: string;
+  strategies: Strategy[];
+};
+
+type Delegation = {
+  delegator?: Member;
+  delegatee?: Member;
+  ratio: number;
+};
+
+type Member = {
+  id?: string;
+  address: string;
+  delegators?: Delegation[];
+  delegatees?: Delegation[];
+};
+
+export async function strategy(
+  space: string,
+  network: string,
+  provider: any,
+  addresses: string[],
+  options: Params = {
+    subgraphUrl: DEFAULT_BACKEND_URL,
+    strategies: []
+  },
+  snapshot: string | number
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+  const block = await provider.getBlock(blockTag);
+
+  const subgraphData: { members: Member[] } = await subgraphRequest(
+    options.subgraphUrl,
+    {
+      members: {
+        __args: {
+          block: { number: block.number }
+        },
```

Comment:
I think this subgraph will return only 1000 members at once by default.

Reply:
You are right! I added a filter to only ask for the addresses needed but also added pagination just in case. Thanks for the heads up